src/helics/cpp98/Federate.hpp

```diff
@@ -374,22 +383,41 @@ class Federate {
     {
         helicsFederateEnterInitializingModeAsync(fed, hThrowOnError());
     }
-    /** called after one of the async calls and will indicate true if an async operation has
-    completed
-    @details only call from the same thread as the one that called the initial async call and will
-    return false if called when no async operation is in flight*/
-    bool isAsyncOperationCompleted() const
-    {
-        // returns int, 1 = true, 0 = false
-        return helicsFederateIsAsyncOperationCompleted(fed, HELICS_NULL_POINTER) > 0;
-    }
+
     /** second part of the async process for entering initializationState call after a call to
     enterInitializingModeAsync if call any other time it will throw an InvalidFunctionCall
     exception*/
     void enterInitializingModeComplete()
     {
         helicsFederateEnterInitializingModeComplete(fed, hThrowOnError());
     }
+
+    /** iterate to the startup up
```

Comment:
```suggestion
    /** iterate to startup during entry to initializing mode
```

I'm not sure if I understood what this was saying.